src/app/api/login/page.tsx

```diff
@@ -0,0 +1,13 @@
+import type { NextPage } from "next";
+import Form from "@/components/common/Form";
+import { ButtonProps } from "@/components/common/types";
+
+const Page: NextPage<ButtonProps> = ({ entry }) => {
```

Comment:
```suggestion
const Page: NextPage = () => {
```
ページコンポーネント自体はpropsを取らないので削除して大丈夫です。

Signup/Singinのフォーム切り替えはページに渡すpropsではなく、アクセスされるパス`/signup`, `/signin`それぞれのページコンポーネント内でそれぞれのFormコンポーネントに渡すようにするイメージです。

Reply:
コンポーネントの親子関係を正しく把握していませんでした！指摘ありがとうございます！